src/WebJobs.Extensions.EasyTables/Config/MobileServiceApiKeyHandler.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.EasyTables
+{
+    /// <summary>
+    /// A <see cref="DelegatingHandler"/> that adds a ZUMO-API-KEY header to every request.
+    /// </summary>
+    internal class MobileServiceApiKeyHandler : DelegatingHandler
```

Comment:
Not anymore; it was removed for Mobile Apps. Now we're (kinda) adding it back for WebJobs. This is what I brought up several times about how we have no auth model that works here. This is our fix for this -- it requires the user to add some code on the server as well.